packages/loopring_v3/contracts/aux/bridge/IBridge.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "./BridgeData.sol";
+
+
+interface IBridge
+{
+     function batchDeposit(
+        address                   from,
+        BridgeTransfer[] calldata deposits
```

Comment:
deposits => transfers?

Reply:
Depends how you look at it. They are done with transfers internally, but they are all separate deposit operations from the caller's point of view.